README.md

```diff
@@ -1,3 +1,6 @@
+# DEPRECATED
+We have replaced `cfssl` with `cert-manager` for CockroachDB certificates. Please refer to [utilitywarehouse/cockroachdb-manifests](https://github.com/utilitywarehouse/cockroachdb-manifests) repository for the up-to-date guides and manifests.
```

Comment:
Lets reword this to: "This is no longer recommended way to deploy CockroachDB, please see <link to cert-manger> for the latest recommended deployment".